tools/dump/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if (NOT "${SPIRV_SKIP_TESTS}" AND TARGET gmock_main)
```

Comment:
Why is the testing not in the "testing" directory?

Reply:
Is it mostly testing the library because that is where most of the functionality is.  I think it should be in https://github.com/KhronosGroup/SPIRV-Tools/tree/main/test/tools.

Reply:
moves to test/tools